classes/PaymentModule.php

```diff
@@ -457,7 +457,7 @@ public function validateOrder(
                     'id_product' => $product['id_product'],
                     'id_product_attribute' => $product['id_product_attribute'],
                     'reference' => $product['reference'],
-                    'name' => $product['name'] . (isset($product['attributes']) ? ' - ' . $product['attributes'] : ''),
+                    'name' => (!empty($product['attributes']) ? ' - ' . $product['attributes'] : ''),
```

Comment:
Why did you remove `$product['name'] . `?

Reply:
I think it's a mistake coming from my review

Reply:
@Seb33300 I fixed it 👍🏻

Reply:
Yes there is a a bug, because it has been removed when I applied the suggestion :-(